### PR TITLE
[press] Added blogs and articles + minor improvements

### PR DESCRIPTION
**Pull request by atb00ker**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -20,10 +20,3 @@ Contents:
    developer/hacking-openwisp-python-django
    developer/google-summer-of-code
    developer/google-code-in
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
```
Comment (atb00ker) [inline index.rst:29]: Dead links.

File: general/press.rst [hunk 2]
```diff
@@ -118,10 +118,38 @@ Blog Posts
...
+- `Building a Javascript Based Configuration UI for OpenWISP <https://medium.com/@gastonche/building-a-javascript-based-configuration-ui-for-openwisp-5eab15088a55>`_ by Nkhoh Gaston Che.
+- `OpenWISP 2 Network Topology <https://medium.com/@rohithasrk/openwisp-2-network-topology-gsoc-17-4765008ccba>`_ by Rohith A. S. R. K.
+- `Google Summer of Code 2017 Django-freeradius <https://delucafiorella2602.wordpress.com/>`_ by Fiorella De Luca.
+- `Raspbian backend for OpenWISP 2 <https://medium.com/@ritwickdsouza/gsoc-openwisp-raspbian-backend-for-openwisp-2-61ff91843362>`_ by Ritwick DSouza.
+
+
+2018 Students
+~~~~~~~~~~~~~
+
+- `OpenWISP IPAM: IP Address Management tool for OpenWISP2 <https://gist.github.com/anurag-ks/75d033c9652c559b065f9cc6320ea707>`_ by Anurag Sharma.
```
Comment (atb00ker) [inline general/press.rst:138]: Rohtith A.S.R.K's work missing from 2018, he didn't write a blog. :-(

File: general/press.rst [hunk 3]
```diff
@@ -118,10 +118,38 @@ Blog Posts
...
+
+2019 Students
+~~~~~~~~~~~~~
+
+- `Dockerization of OpenWISP <https://medium.com/@atb00ker/docker-openwisp-9b2040f03966>`_ by Ajay Tripathi (atb00ker).
```
Comment (atb00ker) [inline general/press.rst:143]: Kutugu's work missing from 2019, he didn't write a blog. :-(

**Comment by ritwickdsouza** [comment]
Thanks @atb00ker !
We can close #114 in favor of this PR then

File: general/press.rst (hunk 2 above)
Comment (nemesisdesign) [reply general/press.rst:138]: There's this one: https://medium.com/@rohithasrk/openwisp-2-network-topology-gsoc-17-4765008ccba

Another interesting gsoc project is here: https://medium.com/@ritwickdsouza/gsoc-openwisp-raspbian-backend-for-openwisp-2-61ff91843362

File: general/press.rst (hunk 3 above)
Comment (nemesisdesign) [reply general/press.rst:143]: here it is: https://medium.com/@zhongliwang48/project-report-netjsongraph-js-library-of-openwisp-d05ef95757d8

**Review by nemesisdesign — COMMENTED** [review]
I merged https://github.com/openwisp/openwisp2-docs/pull/115 since @ritwickdsouza sent it and I appreciate that he took the time to do it, thanks!

File: general/press.rst (hunk 2 above)
Comment (atb00ker) [reply general/press.rst:138]: > There's this one: https://medium.com/@rohithasrk/openwisp-2-network-topology-gsoc-17-4765008ccba

This one is added, it's for 2017, I couldn't find one for 2018.

**Comment by nepython** [comment]
@atb00ker, I found this from gsoc archive https://gist.github.com/rohithasrk/f4dfff366a6b277538c1beff31e1b7c9. It's not a blog post but nevertheless a project project from 2018.

**Comment by nemesisdesign** [comment]
> @atb00ker, I found this from gsoc archive https://gist.github.com/rohithasrk/f4dfff366a6b277538c1beff31e1b7c9. It's not a blog post but nevertheless a project project from 2018.

I think we don't need to add this.

File: general/press.rst [hunk 4]
```diff
@@ -118,12 +118,36 @@ Blog Posts
...
-  Project report <https://medium.com/@nepython/openwisp-monitoring-
-  gsoc-2020-project-report-332441961629>`_
-- `GSOC 2017: adding AirOS support to netjsonconfig <https://edoput.github.io/openwispgsoc/>`_
-- `GSoC 2017 @ OpenWISP: Raspbian backend for OpenWISP 2
-  <https://medium.com/@ritwickdsouza/gsoc-openwisp-raspbian-backend-for-openwisp-2-61ff91843362>`_
+
+Google Summer of Code Blog Posts
+--------------------------------
+
+2017 Students
```
Comment (nemesisdesign) [inline general/press.rst:125]: we should put recent content first, the rest of the section should be organized this way as well, at least the presentations

**Review by nemesisdesign — COMMENTED** [review]
Thanks @atb00ker, could you do what I ask below please?

**Review by nemesisdesign — APPROVED** [review]
Thanks @atb00ker :+1: :clap:

**Pull request merged**